gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -270,14 +285,23 @@ public Integer getChannelAffinity() {
     return channelAffinity;
   }
 
+  /**
+   * The extra header for this context.
+   */
```

Comment:
Should this be a `@BetaApi` as well? This does not seem to be related to streaming. Similar for `getChannelAffinity`

Reply:
I think it should be.